lib/model/FineChannel.js

```diff
@@ -32,4 +33,15 @@ module.exports = class FineChannel extends AbstractChannel {
   get fineness() {
     return this._coarseChannel.fineChannelAliases.indexOf(this.key) + 1;
   }
+
+  get capabilities() {
```

Comment:
Fine channels don't have capabilities by design. If you need them in a plugin, you can still generate them there.